core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/compare/CompareSetResult.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2020 B2i Healthcare. All rights reserved.
+ *******************************************************************************/
+package com.b2international.snowowl.core.compare;
+
+import java.util.Set;
+
+import com.google.common.collect.ListMultimap;
+/**
+ * @since 7.8
+ */
+public class CompareSetResult <T> {
+	Set<T> addedMembers;
+	Set<T> removedMembers;
+	ListMultimap<T, T> changedMembers;
+	
+	public CompareSetResult(Set<T> addedMembers, Set<T> removedMembers, ListMultimap<T, T> changedMembers) {
+		this.addedMembers = addedMembers;
```

Comment:
Make sure you defensive copy the incoming Collection-like parameters with the `Immutable*.copyOf()` equivalent.

Reply:
Done